pom.xml

```diff
@@ -11,9 +11,9 @@
     <relativePath/> <!-- lookup parent from repository -->
   </parent>
 
-  <groupId>org.molgenis</groupId>
+  <groupId>com.github.molgenis</groupId>
```

Comment:
why this change?

Reply:
this is related to the jitpack construction to use github releases as maven dependecies. We're looking into fixing this with a different construction for this behaviour (e.g. github packages)